### ISSUE TITLE
library not found for -lPods

### ISSUE THREAD

**Issue by wwrrss** [googlesamples/cardboard-ios#17]
I'm getting this error when i build the VideoWidgetDemo using CocaPods 1.0.0, Xcode 7.3, OS X 10.11.4:  

`ld: library not found for -lPods
clang: error: linker command failed with exit code 1 (use -v to see invocation)`

`Ld /Users/william/Library/Developer/Xcode/DerivedData/VideoWidgetDemo-gciobuvjychjulgwraiodifnopto/Build/Products/Debug-iphonesimulator/VideoWidgetDemo.app/VideoWidgetDemo normal x86_64
    cd /Users/william/Documents/ios/cardboard-ios/CardboardSamples/VideoWidgetDemo
    export IPHONEOS_DEPLOYMENT_TARGET=9.2
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.3.sdk -L/Users/william/Library/Developer/Xcode/DerivedData/VideoWidgetDemo-gciobuvjychjulgwraiodifnopto/Build/Products/Debug-iphonesimulator -L/Users/william/Documents/ios/cardboard-ios/CardboardSamples/VideoWidgetDemo/Pods/CardboardSDK/Libraries -F/Users/william/Library/Developer/Xcode/DerivedData/VideoWidgetDemo-gciobuvjychjulgwraiodifnopto/Build/Products/Debug-iphonesimulator -filelist /Users/william/Library/Developer/Xcode/DerivedData/VideoWidgetDemo-gciobuvjychjulgwraiodifnopto/Build/Intermediates/VideoWidgetDemo.build/Debug-iphonesimulator/VideoWidgetDemo.build/Objects-normal/x86_64/VideoWidgetDemo.LinkFileList -Xlinker -rpath -Xlinker @executable_path/Frameworks -mios-simulator-version-min=9.2 -Xlinker -no_deduplicate -Xlinker -objc_abi_version -Xlinker 2 -ObjC -lCardboardSDK -lc++ -framework AVFoundation -framework AudioToolbox -framework CoreGraphics -framework CoreMedia -framework CoreMotion -framework CoreVideo -framework GLKit -framework MediaPlayer -framework OpenGLES -framework QuartzCore -fobjc-arc -fobjc-link-runtime -lPods -lPods-VideoWidgetDemo -Xlinker -dependency_info -Xlinker /Users/william/Library/Developer/Xcode/DerivedData/VideoWidgetDemo-gciobuvjychjulgwraiodifnopto/Build/Intermediates/VideoWidgetDemo.build/Debug-iphonesimulator/VideoWidgetDemo.build/Objects-normal/x86_64/VideoWidgetDemo_dependency_info.dat -o /Users/william/Library/Developer/Xcode/DerivedData/VideoWidgetDemo-gciobuvjychjulgwraiodifnopto/Build/Products/Debug-iphonesimulator/VideoWidgetDemo.app/VideoWidgetDemo`

**Comment by sanjayc77** [comment]
Are you opening VideoWidgetDemo.xcworkspace instead of VideoWidgetDemo.xcodeproject in Xcode? Cocoapods requires using the .xcworkspace project file.

Another trick to try, just run 'pod try CardboardSDK' on the command line. That should open the video demo app.

**Comment by wwrrss** [comment]
Yes, i opened VideoWidgetDemo.xcworkspace, i ran the Panorama and TreasureHunt demos  with no problem. When i run 'pod try CardboardSDK' i get this message: 

`Williams-MacBook-Pro:VideoWidgetDemo william$ pod try CardboardSDK
Updating spec repositories

Trying CardboardSDK
Performing CocoaPods Installation
[!] The dependency `CardboardSDK` is not used in any concrete target.
Williams-MacBook-Pro:VideoWidgetDemo william$ 
`

I'm using CocoaPods 1.0.0. and according to the Cocoa documentation you have to define a target in your pod file. Maybe this is the problem?

**Comment by sanjayc77** [comment]
We started seeing this recently too. Try:

target 'VideoWidgetDemo' do
  pod 'CardboardSDK'
end

**Comment by wwrrss** [comment]
I have tried that too but with no lucky, as i said before i ran with no problems the Panorama demo and Treasure Hunt . 

My pod file content:

target 'VideoWidgetDemo' do

pod 'CardboardSDK'

end

**Comment by wwrrss** [comment]
I also found this: 
Target 'Pods-VideoWidgetDemo' of project 'Pods' was rejected as an implicit dependency for 'libPods-VideoWidgetDemo.a' because its architectures 'x86_64' didn't contain all required architectures 'i386 x86_64

**Comment by sanjayc77** [comment]
From another developer:

"Another issue I noticed was that when you try building Stars or VideoWidget in Xcode, there would be an error: "library not found for -lPods". I was thinking it might be an artifact from the old version of Cocoapods. I removed the file link in XCode, under "Linked Frameworks and Libraries", and it built successfully. I also found a reference online documenting the same issue: http://www.mokacoding.com/blog/upgrading-podfile/"
